healpix/high_level.py

```diff
@@ -328,7 +328,7 @@ def cone_search_skycoord(self, skycoord, radius, approximate=False):
         skycoord = skycoord.transform_to(self.frame)
         representation = skycoord.represent_as(UnitSphericalRepresentation)
         lon, lat = representation.lon, representation.lat
-        return self.cone_search_lonlat(lon, lat, radius, self.nside)
+        return self.cone_search_lonlat(lon, lat, radius, approximate)
```

Comment:
Could you use the keyword argument notation? (``approximate=approximate``). This will avoid future issues.

Reply:
Do you mean just for the last argument, or pass all arguments along as keyword arguments?
I usually pass everything as keyword arguments, but either way is fine with me.

Reply:
I meant specifically ``approximate`` since it's an actual keyword argument in ``cone_search_lonlat``.

We could indeed expand to always specifying the argument name for all args, but then we should do it consistently. I think it's mostly important here since ``approximate`` is a kwarg and therefore API-wise we could choose to move it position-wise.

Reply:
OK, let's use the pattern to pass keyword arguments if the called function uses a keyword argument.